### DISCUSSION TITLE
Allow using PostgreSQL for most use cases that currently uses SQS, SNS and Redis.

### DISCUSSION THREAD

**Discussion by seivan** [encoredev/encore#2313] [Suggestions]
Could pretty much replace `SQS`, `SNS` and even `Redis` and still maintain feature parity. Might not have the same performance measurements but getting rid of extra infra and services would fit probably fit most people. 

I don't see how it would be a problem if it was a toggle in config. 

Just less stuff to manage and pay for.

**Comment by simon-johansson** [comment]
Converting this into a Suggestions discussion, we are trying to keep issues for bug reports.
